src/hotspot/cpu/ppc/macroAssembler_ppc.inline.hpp

```diff
@@ -236,6 +236,30 @@ inline bool MacroAssembler::is_bc_far_variant3_at(address instruction_addr) {
          is_endgroup(instruction_2);
 }
 
+// set dst to -1, 0, +1
```

Comment:
Added this comment:
```
// set dst to -1, 0, +1 as follows: if CCR0bi is "greater than", dst is set to 1,
// if CCR0bi is "equal", dst is set to 0, otherwise it's set to -1.
```

Reply:
Looks good!